tools/owlbot-postprocess.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -e
+
+# Note: we could do various parts of this globally from the top level,
+# but doing it on a per-API level means we don't need to worry about
+# overly long command lines etc
+
+cd owlbot-staging
+for api in *
+do
+  echo "Processing $api"  
+  # Remove existing generated code from target directory
+  rm -rf ../apis/*/*.g.cs
```

Comment:
Should this remove the code for just the current api?
Also, it's only removing the .g.cs, but what about the .cs only that you have to rename later?
There might be something I'm missing here though.

Reply:
Eesh! Yes it should. That first `/*` should be `/$api`. Doh!